content/en/docs/demo/services/frontend.md

```diff
@@ -178,10 +178,9 @@ const FrontendTracer = async () => {
     resource: new Resource({
       [SEMRESATTRS_SERVICE_NAME]: process.env.NEXT_PUBLIC_OTEL_SERVICE_NAME,
     }),
+    spanProcessors: [new SimpleSpanProcessor(new OTLPTraceExporter())],
```

Comment:
This file should be the snippet of [`FrontendTracer.ts`](https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/frontend/utils/telemetry/FrontendTracer.ts#L4-L69).
But it seems it is already out-of-date, so I won't block this PR.

I've created the issue https://github.com/open-telemetry/opentelemetry-demo/issues/2080 to follow-up on the Demo repo.